compiler/qsc_parse/src/item.rs

```diff
@@ -561,12 +562,31 @@ fn parse_import_or_export(s: &mut ParserContext) -> Result<ImportOrExportDecl> {
 
 fn parse_import_or_export_item(s: &mut ParserContext) -> Result<ImportOrExportItem> {
     // an import item is a path followed by an optional alias,
-    let path = *(path(s)?);
+    let mut idents = Vec::default();
+    let mut is_glob = false;
+    loop {
+        idents.push(*ident(s)?);
+        if s.peek().kind == TokenKind::Dot {
+            s.advance();
+        }
+        if s.peek().kind == TokenKind::ClosedBinOp(ClosedBinOp::Star) {
```

Comment:
I leave the solution up to your judgment, but it sort of bothers me that `*` is still considered a `ClosedBinOp`. I don't know if this has any practical implications in the code, or if it's just a naming thing.